recipes/rapidjson/all/conandata.yml

```diff
@@ -15,3 +15,7 @@ sources:
   "cci.20200410":
     url: "https://github.com/Tencent/rapidjson/archive/8f4c021fa2f1e001d2376095928fc0532adf2ae6.zip"
     sha256: e6fc99c7df7f29995838a764dd68df87b71db360f7727ace467b21b82c85efda
+patches:
+  "1.1.0":
+    - patch_file: "patches/0001-gcc14.patch"
+      patch_description: "Fix for GCC 14 compiler error in document.h."
```

Comment:
Has this been reported upstream?

Reply:
1.1.0 is actually from 2016, I'm surprised we support it to begin with (though it seems it's used at least). The patch probably isn't relevant by now. Would be worth checking to see if the latest works with GCC 14 though.

Reply:
We need this for our CI.  I patched the recipe and the source.  We have a legacy app still using 1.1.0 which was the last official release.

Reply:
In fact, this is fixed origin/master.  There's just no released fix for it.

Reply:
I just changed the patch to do what was done in rapid json's main.  They just removed the `=` operator altogether.